python/marvin/contrib/vacs/mangafirefly.py

```diff
@@ -0,0 +1,84 @@
+# !usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Licensed under a 3-clause BSD license.
+#
+# @Author: Brian Cherinka
+# @Date:   2018-10-11 17:51:43
+# @Last modified by:   Brian Cherinka
+# @Last Modified time: 2018-11-29 17:23:15
+
+from __future__ import print_function, division, absolute_import
+
+import numpy as np
+import astropy
+import astropy.units as u
+import marvin.tools
+from marvin.tools.quantities.spectrum import Spectrum
+from marvin.utils.general.general import get_drpall_table
+from marvin.utils.plot.scatter import plot as scatplot
+
+from .base import VACMixIn
+
+
+class FIREFLYVAC(VACMixIn):
+    """Provides access to the MaNGA-FIREFLY VAC.
+
+    VAC name: FIREFLY
+
+    URL: 
+
+    Description: Returns integrated and resolved stellar population parameters fitted by FIREFLY
+
```

Comment:
Can you change the URL link to point to the DR15 documentation page for your VAC?